INSTALL.rst

```diff
@@ -470,31 +470,34 @@ Custom Targets
 
 All custom test targets build and run a preconfigured set of tests.
 
-- ``make check [-jN] [ARGS=-jN]``
+- ``cmake --build . -t check [-jN]``
   The default build-and-test target for cvc5, builds and runs all examples,
   all system and unit tests, and regression tests from levels 0 to 2.
 
-- ``make systemtests [-jN] [ARGS=-jN]``
+- ``cmake --build . -t systemtests [-jN]``
   Build and run all system tests.
 
-- ``make units [-jN] [ARGS=-jN]``
+- ``cmake --build . -t units [-jN]``
   Build and run all unit tests.
 
-- ``make regress [-jN] [ARGS=-jN]``
+- ``cmake --build . -t regress [-jN]``
   Build and run regression tests from levels 0 to 2.
 
-- ``make runexamples [-jN] [ARGS=-jN]``
+- ``cmake --build . -t runexamples [-jN]``
   Build and run all examples.
 
-- ``make coverage-test [-jN] [ARGS=-jN]``
+- ``cmake --build . -t coverage-test [-jN]``
```

Comment:
Are these still accurate? I can't find any evidence that the `systemtests`, `runexamples`, or `coverage-test` targets still exist.

Reply:
No, that is not accurate. These are obsolete targets and should be removed from the `INSTALL.rst` file. Thank you for pointing this out.